content/en/docs/concepts/storage/volumes.md

```diff
@@ -311,6 +311,10 @@ third party storage driver instead.
 The `gitRepo` volume type is deprecated. To provision a container with a git repo, mount an
 [EmptyDir](#emptydir) into an InitContainer that clones the repo using git, then mount the
```

Comment:
You might also want to make this change:
```suggestion
[`emptyDir`](#emptydir) volume into an [init container](/docs/concepts/workloads/pods/init-containers/) that
clones the repo using Git, then mount the
```

:thought_balloon: Another option is to use a sidecar that remains running, for example with
https://github.com/kubernetes/git-sync